client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx

```diff
@@ -580,7 +582,7 @@ export const ApplicationsTable: React.FC = () => {
 
     const row = selectedRows[0];
     const assessment = getApplicationAssessment(row.id!);
-    if (!assessment) {
+    if (!assessment && !reviewAssessmentSetting.data) {
```

Comment:
Could we destructure here and alias the setting to a name that would be self documenting?

Something like:

``` 
  const {data: requireAssessBeforeReview} = useSetting("review.assessment.required");
```
or something to that effect.